app/src/main/java/com/mobile/tandil/kotlinbaseproject/mvvm/viewmodel/MainDialogViewModel.kt

```diff
@@ -0,0 +1,13 @@
+package com.mobile.tandil.kotlinbaseproject.mvvm.viewmodel
+
+import androidx.lifecycle.ViewModel
+import com.mobile.tandil.kotlinbaseproject.mvvm.model.MainDialogModel
+
+class MainDialogViewModel : ViewModel() {
```

Comment:
the model should be passed as parameter in the viewModel constructor

Reply:
done